working/base-interface-final/feature-specification.md

```diff
@@ -477,11 +549,16 @@ It is a compile-time error to:
     _own_ interface that could then be implemented without inheriting the
     concrete implementation from the supertype.*
 
-*   Mix in a class marked `sealed`, `base`, `interface`, or `final`. *We want to
-    eventually move away from classes as mixins. We don't want to break existing
-    uses of classes as mixins but since no existing code is using these
-    modifiers, we can prevent classes using those modifiers from also being used
-    as mixins.*
+*   Mix in a class not marked `mixin` outside of the library where it is
+    defined, unless the class declaration is in a library whose language version
+    is older than the version this feature ships in.
+
```

Comment:
Should we add

  * Mix in a class with a superclass other than `Object`, or which declares a generative constructor.

(which only applies to mixin applications of a non-`mixin` `class` inside the same library. Or do we consider that covered by the existing spec.)

Reply:
Added a parenthetical to clarify.